src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -0,0 +1,27 @@
+const MORPH_SUFFIX = 'morph_env=test';
```

Comment:
Are we ok with this being in simorgh?

Reply:
This is being removed as we speak. It was to overcome the issue with mismatching our other applications, but that's now been resolved.